docs/service-discovery/dns.md

```diff
@@ -0,0 +1,46 @@
+# DNS Service Discovery
+
+DNS is a name resolution protocol used to determine IP addresses for hostnames.
+That makes it a natural fit for service discovery. Among others, Consul and AWS Cloud Map provide DNS APIs for service discovery.
+
+This page explains how Stork can use DNS to handle the service discovery.
+
+By default, DNS service discovery uses _SRV_ records to discover service locations. The target of the _SRV_ record is
+resolved to IP by querying for _A_ and _AAAA_ records.
+
+## Dependency
+
+To use the DNS service discovery, you need to add the Stork DNS Service Discovery provider dependency to your project:
+
+```xml
+<dependency>
+    <groupId>io.smallrye.stork</groupId>
+    <artifactId>stork-service-discovery-dns</artifactId>
+    <version>{{version.current}}</version>
+</dependency>
+```
+
+## Configuration
+
+Next, set the service discovery `type` to `DNS`.
```

Comment:
Here you use uppercases, but you use lowercases in the example. Should we always use lowercase?